lightning/src/ln/channel.rs

```diff
@@ -1719,6 +1722,30 @@ impl FundingScope {
 	}
 }
 
+/// Info about a pending splice, used in the pre-splice channel
+#[cfg(splicing)]
+#[derive(Clone)]
+struct PendingSplice {
+	pub our_funding_contribution: i64,
+}
+
+#[cfg(splicing)]
+impl PendingSplice {
+	#[inline]
+	fn add_checked(base: u64, delta: i64) -> u64 {
+		if delta >= 0 {
+			base.saturating_add(delta as u64)
+		} else {
+			base.saturating_sub(delta.abs() as u64)
+		}
+	}
+
+	/// Compute the post-splice channel value from the pre-splice values and the peer contributions
+	pub fn compute_post_value(pre_channel_value: u64, our_funding_contribution: i64, their_funding_contribution: i64) -> u64 {
+		Self::add_checked(pre_channel_value, our_funding_contribution.saturating_add(their_funding_contribution))
+	}
```

Comment:
Looks like this isn't needed in this PR anymore. Same with `add_checked`.

Reply:
Indeed. Interesting, `cargo t -p lightning <pattern>` did not show these unused warnings, while `cargo t -p lightning` does (not clear to me why).
These are removed now.

Reply:
You'll need to use `RUSTFLAGS="--cfg=splicing" cargo +1.63.0 test -p lightning`

Reply:
I've been using `splicing` flag, but interestingly the difference is between:
```
RUSTFLAGS="--cfg=splicing" cargo t -p lightning splic
```
and
```
RUSTFLAGS="--cfg=splicing" cargo t -p lightning
```
The former did not show the unused warning, the latter did. Oh, well. That's why we have CI's :)